web/assets/css/bundles_list.css

```diff
@@ -0,0 +1,73 @@
+.bundles-list {
+    overflow: hidden;
+}
+
+.bundles-list-header {
+    border-top: 1px solid rgb(68,68,68);
+}
+
+    .bundles-list-header .bundles-list-hello-message {
+        padding: 50px 0 0 0;
+        font-size: 16px;
+    }
+
+    .bundles-list-header h2 {
```

Comment:
There is no `h2` in the code.
(before my comment regarding changing `h4` to `h2`)